tests/real/heart2.expected

```diff
@@ -1 +1 @@
-{"mappings":{"mouse":"f","resolution":"y","time":"v"},"shaders":{"tests/real/heart2.frag":"uniform float v;uniform vec2 y;uniform vec4 f;void main(){vec2 f=(2.*gl_FragCoord.xy-y)/y.y;float g=mod(v,2.)/2.,a=pow(g,.2)*.5\u002B.5;a-=a*.2*sin(g*6.2831*5.)*exp(-g*6.);f*=vec2(.5,1.5)\u002Ba*vec2(.5,-.5);g=atan(f.x,f.y)/3.141593;a=length(f);g=abs(g);g=(13.*g-22.*g*g\u002B10.*g*g*g)/(6.-5.*g);gl_FragColor=vec4(step(a,g)*pow(1.-a/g,.25),0,0,1);}"}}
+{"mappings":{"mouse":"f","resolution":"y","time":"v"},"shaders":{"tests/real/heart2.frag":"uniform float v;uniform vec2 y;uniform vec4 f;void main(){vec2 f=(2.*gl_FragCoord.xy-y)/y.y;float g=mod(v,2.)/2.,a=pow(g,.2)*.5+.5;a-=a*.2*sin(g*6.2831*5.)*exp(-g*6.);f*=vec2(.5,1.5)+a*vec2(.5,-.5);g=atan(f.x,f.y)/3.141593;a=length(f);g=abs(g);g=(13.*g-22.*g*g+10.*g*g*g)/(6.-5.*g);gl_FragColor=vec4(step(a,g)*pow(1.-a/g,.25),0,0,1);}"}}
```

Comment:
Why was the json library quoting + and è, and shouldn't we quote them too?

Reply:
Maybe security risks 🤔 
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/character-encoding#serialize-all-characters

Reply:
Ok, I guess we don't really care about those.